source/Fleet.h

```diff
@@ -18,6 +18,7 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #include "Variant.h"
 #include "WeightedList.h"
 
+#include <cstdint>
```

Comment:
I believe you commented that Variant needed this since Strength returns int64_t. I spotted that Fleet also has a Strength method that returns int64_t, but it lacked this include.